### PR TITLE
Temporarily skip unstable tests

### PR DESCRIPTION
**Pull request by ashtul**

### CODE REVIEW

**Review by rafie — DISMISSED** [review]

**Comment by rafie** [comment]
@ashtul We should probably have the `UNSTABLE_TESTS = os.getenv('UNSTABLE_TESTS', '0') == '1'` defined in `includes.py` instead of in every test file.

File: tests/pytests/test_config.py [hunk 1]
```diff
@@ -153,7 +154,8 @@ def test_arg_true(arg_name):
         env.stop()
 
     test_arg_true('NOGC')
-    test_arg_true('SAFEMODE')
+    if not is_unstable():
+        test_arg_true('SAFEMODE')
```
Comment (MeirShpilraien) [inline tests/pytests/test_config.py:158]: Why is it considered unstable?

File: tests/pytests/test_ext.py [hunk 2]
```diff
@@ -26,6 +27,9 @@
     print subprocess.call(args)
 
 def testExt():
+    if not is_unstable():
```
Comment (MeirShpilraien) [inline tests/pytests/test_ext.py:30]: Also this one, why its unstable?

File: tests/pytests/test_json.py [hunk 3]
```diff
@@ -95,8 +93,12 @@ def testHandleUnindexedTypes(env):
     # TODO: Ignore and resume indexing when encountering an Object/Array/null
     # TODO: Except for array of only scalars which is defined as a TAG in the schema
     # ... FT.CREATE idx SCHEMA $.arr TAG
-    if not UNSTABLE_TESTS:
+    if not is_unstable():
+        raw_input('stop')
```
Comment (MeirShpilraien) [inline tests/pytests/test_json.py:97]: Remove? Also why its unstable?

File: tests/pytests/test_json.py [hunk 4]
```diff
@@ -444,7 +446,7 @@ def testNumeric(env):
         .equal([1L, 'doc:1', ['flt', '9.72']])
 
 def testLanguage(env):
-    if not UNSTABLE_TESTS:
+    if not is_unstable():
```
Comment (MeirShpilraien) [inline tests/pytests/test_json.py:449]: Same question

**Comment by ashtul** [comment]
@MeirShpilraien we don't know why they are unstable. These are build bugs and until they are fixed, these failing tests distract the developer.

File: tests/pytests/common.py [hunk 5]
```diff
@@ -67,3 +68,7 @@ def check_server_version(env, ver):
     if version.parse(server_v) >= version.parse(ver):
         return True
     return False
+
+UNSTABLE_TESTS = os.getenv('UNSTABLE_TESTS') == '1'
```
Comment (rafie) [inline tests/pytests/common.py:72]: `UNSTABLE_TESTS = os.getenv('UNSTABLE_TESTS', '0') == '1'` should be good enough. Let's just place it in `includes.py` so it's available in all test .py files.

**Comment by rafie** [comment]
@MeirShpilraien I'm handling the "unstable" failures on a different thread. As those are infrastructure failures, let's skip them until I find a solution to avoid unnecessary noise.

**Pull request closed without merging**